scripts/rs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "procdockerstatsd_rs"
+version = "1.0.0"
+edition = "2024"
+description = "Daemon to collect process and Docker stats and push them to Redis"
+license = "MIT"
+authors = ["Feng Pan"]
+
+[dependencies]
+redis = "0.23"
```

Comment:
we have wrapped the swss-common rust APIs here: https://github.com/sonic-net/sonic-dash-ha/tree/master/crates/swss-common, so I wonder if we can use it instead of using redis package directly.